src/jaxsim/rbda/contacts/visco_elastic.py

```diff
@@ -925,7 +928,6 @@ def integrate_data_with_average_contact_forces(
         # Create the data at the final time.
         with data.editable(validate=True) as data_tf:
             data_tf: js.data.JaxSimModelData
```

Comment:
Probably this can be removed
```suggestion
```

Reply:
Right, done in https://github.com/ami-iit/jaxsim/pull/262/commits/14d84325b489c81c1e38d64d96d0e2ba2252c50f, thanks!